best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
> ... as nobody seems to agree on what they mean.

I think we could attempt to establish what they mean and agree on some guidelines. Could we use [Service Layer](http://martinfowler.com/eaaCatalog/serviceLayer.html) for some ideas?

> Thoughts on using "method object" instead?

Would these go under `app/methods`, or would we just describe service objects in terms of method objects? I haven't seen `app/methods` out in the wild. The purpose of this PR was focused solely on getting some conventions/consistency around service objects that have been showing up in codebases in the last couple of years.